easybuild/easyblocks/s/score_p.py

```diff
@@ -85,6 +86,29 @@ def configure_step(self, *args, **kwargs):
                 comp_opts[toolchain.NVHPC] = 'pgi'
 
             comp_fam = self.toolchain.comp_family()
+
+            # Since intel-compilers 2022.2.0 it uses the icx/icpx compilers, AfS projects support them
+            # since these version
+            oneapi_since = {
+                'Score-P': '8.0',
+                'Scalasca': '2.6.1',
+                'OTF2': '3.0.2',
+                'CubeWriter': '4.8',
+                'CubeLib': '4.8',
+                'CubeGUI': '4.8',
+            }
+            if comp_fam is toolchain.INTELCOMP \
+               and (self.toolchain.options.get('oneapi', False) or self.toolchain.options.get('oneapi_c_cxx', False)):
+                if LooseVersion(self.version) < LooseVersion(oneapi_since.get(self.name, '0')):
+                    # AfS packages do not yet support oneAPI compilers
+                    raise EasyBuildError("Support to build with Intel oneAPI family started with " +
+                                         f"{self.name} {oneapi_since.get(self.name, '0')} but not {self.version}")
+                if not self.toolchain.options.get('oneapi_fortran', False):
```

Comment:
This will fail in default scenarios: For "intel-compilers >= 2022.2.0" only `oneapi_c_cxx` is enabled by default, but not `oneapi_fortran`. I opened [an issue](https://github.com/easybuilders/easybuild-framework/issues/4455) asking for clarification/change